app/src/components/TeamPayroll/SalaryStats.js

```diff
@@ -3,15 +3,23 @@ import EquityOption from '../EquityOption'
 import PaidSalariesChart from './PaidSalariesChart'
 import SalaryBurnRate from './SalaryBurnRate'
 import Split from '../Split'
+import EditEquity from '../../panels/EditEquity'
 
-function SalaryStats() {
+function SalaryStats({ editEquityOptionPanel, onEditEquityOption }) {
   return (
     <Split>
       <SalaryBurnRate />
-      <EquityOption readOnly={false} />
+      <EquityOption
+        readOnly={false}
+        editEquityOptionPanel={editEquityOptionPanel}
+      />
+      <EditEquity
+        editEquityOptionPanel={editEquityOptionPanel}
```

Comment:
We could use `panelState` instead of `editEquityOptionPanel` since it would be a local prop.

```suggestion
        panelState={editEquityOptionPanel}
```